src/FunctionApps/SilverTransforms/Transform/fhir.py

```diff
@@ -0,0 +1,40 @@
+import io
+import json
+
+from typing import Iterable
+
+from azure.storage.blob import ContainerClient
+
+
+def get_patient_records(conn_str: str, container_name: str) -> Iterable[dict]:
+    """
+    Given an bucket connection string and a container name,
```

Comment:
Nittiest possible nit: Should be `a`, not `an` 😂

Reply:
:heavy_check_mark: thanks!